sky/cli.py

```diff
@@ -3063,12 +3063,30 @@ def _output():
                 return
 
         # Show detailed accelerator information
+        accelerator_split = accelerator_str.split(':')
+        if len(accelerator_split) > 2:
+            raise click.UsageError(
+                f'Invalid accelerator string {accelerator_str}. '
+                'Expected format: <accelerator_name>[:<quantity>].')
+        if len(accelerator_split) == 2:
+            name = accelerator_split[0]
+            # Check if quantity is valid
+            try:
+                quantity = int(accelerator_split[1])
+            except ValueError as invalid_quantity:
+                raise click.UsageError(
+                    f'Invalid accelerator quantity {accelerator_split[1]}. '
+                    'Expected an integer.') from invalid_quantity
```

Comment:
Got all output for V100:0:

```
(base) ➜  ~ sky show-gpus V100:0
*NOTE*: for most GCP accelerators, INSTANCE_TYPE == (attachable) means the host VM's cost is not included.

GPU   QTY  CLOUD   INSTANCE_TYPE       DEVICE_MEM  vCPUs  HOST_MEM  HOURLY_PRICE  HOURLY_SPOT_PRICE  REGION
V100  1    AWS     p3.2xlarge          16GB        8      61GB      $ 3.060       $ 0.918            ap-northeast-1
V100  4    AWS     p3.8xlarge          16GB        32     244GB     $ 12.240      $ 3.672            ap-northeast-1
V100  8    AWS     p3.16xlarge         16GB        64     488GB     $ 24.480      $ 7.344            ap-northeast-1
V100  1    Azure   Standard_NC6s_v3    -           6      112GB     $ 3.060       $ 1.024            centralus
V100  2    Azure   Standard_NC12s_v3   -           12     224GB     $ 6.120       $ 2.048            centralus
V100  4    Azure   Standard_NC24rs_v3  -           24     448GB     $ 13.460      $ 4.506            centralus
V100  4    Azure   Standard_NC24s_v3   -           24     448GB     $ 12.240      $ 4.097            centralus
V100  1    GCP     (attachable)        -           -      -         $ 2.480       $ 0.740            us-central1
V100  2    GCP     (attachable)        -           -      -         $ 4.960       $ 1.480            us-central1
V100  4    GCP     (attachable)        -           -      -         $ 9.920       $ 2.960            us-central1
V100  8    GCP     (attachable)        -           -      -         $ 19.840      $ 5.920            us-central1
V100  1    IBM     gx2-8x64x1v100      -           -      64GB      $ 2.497       -                  au-syd
V100  8    Lambda  gpu_8x_v100         16GB        92     448GB     $ 4.400       -                  asia-northeast-1

GPU        QTY  CLOUD  INSTANCE_TYPE       DEVICE_MEM  vCPUs  HOST_MEM  HOURLY_PRICE  HOURLY_SPOT_PRICE  REGION
V100-32GB  8    AWS    p3dn.24xlarge       32GB        96     768GB     $ 31.212      $ 9.364            ap-northeast-1
V100-32GB  8    Azure  Standard_ND40rs_v2  -           40     672GB     $ 22.032      $ 3.530            eastus
```

Can we:
1. Trigger this error message when quantity==0.
2. Update this message to `Expected a non-zero integer.`

Reply:
Is it better to have this as an output message or as a click.UsageError?

Reply:
Now triggers as error message